src/sage/repl/preparse.py

```diff
@@ -1189,6 +1190,14 @@ def preparse_numeric_literals(code, extract=False, quotes="'"):
         sage: preparse_numeric_literals('000042')
         'Integer(42)'
 
+    Cheeck that :issue:`40179` is fixed::
```

Comment:
```suggestion
    Cheeck that :issue:`40179` is fixed::

```
This needs a blanking after it.

Reply:
Also, "Ch**ee**ck"?

Side note you ( @soon-haari ) can build documentation and view it locally (the doc preview CI is broken). If you're using meson the documentation is unfortunately sparse but there's like https://github.com/sagemath/sage/pull/39973 (documentation contribution is welcome too)